Multisig/Data/Services/Utils/GSError.swift

```diff
@@ -183,8 +183,8 @@ enum GSError {
 
     struct EntityNotFound: DetailedLocalizedError {
         let description = "HTTP 404 Not Found"
-        let reason = "Safe not found."
-        let howToFix = "Please check that the Safe exists on the blockchain"
+        let reason = "Safe Account not found."
```

Comment:
I know we already agreed on this but, Shouldn't "Account" be lower case here?

Reply:
no, it would be inconsistent with the rest of the app and the current state of the web implementation.